data_safe_haven/infrastructure/project_manager.py

```diff
@@ -273,6 +279,9 @@ def evaluate(self, result: str) -> None:
             msg = "Pulumi operation failed."
             raise DataSafeHavenPulumiError(msg)
 
+    def from_ansi(self) -> Callable[[str], None]:
+        return partial(from_ansi, logger=self.logger)
```

Comment:
Could be worth changing this method name, since in this place we're not printing a message "from ansi" but instead sending a message "to logger" or similar?

Reply:
Done in 6cd29be9